<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>631</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Program</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>pattern</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>directory</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>directory of result </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>delete entering files</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>rewrite new files</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>timer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>binary operation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>value</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPattern">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDirectory">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditResultDirectroy">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDeleteEntering">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxRewrite">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>150</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>170</y>
       <width>118</width>
       <height>26</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>HH:mm:ss</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonXOR">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>200</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>XOR</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonAND">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>200</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>AND</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonOR">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>200</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>OR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditValue">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>220</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRrocedure">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>300</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Procedure</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditDebug">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>340</y>
       <width>521</width>
       <height>70</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
